spec/Features/food_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Index Page' do
+  before(:each) do
+    @user = create(:user)
+    @user.confirm
+    login_as(@user, scope: :user)
+    @food = create(:food, user_id: @user.id)
+  end
+
+  it 'can see the add food button' do
```

Comment:
- [ ] @Team Great work trying to fix the test errors, however some of your tests are still failing, kindly consider fixing these tests to ensure all tests are passing. 
![image](https://github.com/Chimoneg27/Rails-Recipe-App/assets/61408860/e60487e1-ac53-4253-bd9c-896864f87465)

Reply:
@danmainah they pass on my pc

Reply:
@danmainah they pass on my pc